README.md

```diff
@@ -29,14 +29,14 @@ Once unzipped, right-click the font file and click "Install for all users". This
 For more details and app-specific instructions, [please check the wiki](https://github.com/microsoft/cascadia-code/wiki/Installing-Cascadia-Code). 
 
 # Get involved
-Instructions on how to modify and submit an update to the Cascadia Code source is [available in the wiki](https://github.com/microsoft/cascadia-code/wiki/Modifying-Cascadia-Code).
+Instructions on how to modify and submit an update to the Cascadia Code source are [available in the wiki](https://github.com/microsoft/cascadia-code/wiki/Modifying-Cascadia-Code).

```

Comment:
In this case, "source" is singular. I suppose we could say "sources" since there are multiple UFOs, but if we're referring to the github repository as a whole, singular is OK.